js/integration-tests/solidity-tests/contracts/ContractEnvironment.t.sol

```diff
@@ -20,8 +17,4 @@ contract ContractEnvironmentTest is DSTest {
         assertEq(block.number, 23, "block number is incorrect");
         assertEq(block.timestamp, 45, "timestamp is incorrect");
     }
-    
-    function testContextIsTest() public {
-        assertEq(vm.isContext(Vm.ExecutionContext.Test), true);
-    }
 }
```

Comment:
I removed this test case, because the execution context doesn't seem to be exposed from `forge-std` and it's unnecessary anyway: there is an identical Rust integration test.